### ISSUE TITLE
t/01-basic.t started to fail

### ISSUE THREAD

**Issue by eserte** [cromedome/Dancer2-Plugin-Minion#1]
On my smoker systems:
```
PERL_DL_NONLAZY=1 "/opt/perl-5.31.11/bin/perl5.31.11" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
[TestApp:25786] error @2020-06-18 20:03:33> Route exception: Task "foo" missing at /home/cpansand/.cpan/build/2020061820/Dancer2-Plugin-Minion-0.1.0-0/blib/lib/Dancer2/Plugin/Minion.pm line 37. in /opt/perl-5.31.11/lib/site_perl/5.31.11/Dancer2/Core/App.pm l. 1493

#   Failed test 'GET /'
#   at t/01-basic.t line 14.
# 500
# Internal Server Error

#   Failed test 'Added a new task to Minion'
#   at t/01-basic.t line 15.
#          got: "<!DOCTYPE html>\x{0a}<html lang="en">\x{0a}<head>\x{0a}  <meta ch"...
#       length: 491
#     expected: "OK - Task Added"
#       length: 15
#     strings begin to differ at char 1 (line 1 column 1)
# Looks like you failed 2 tests of 7.
t/01-basic.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/7 subtests 
```

Statistical analysis has to candidates: newer versions of Minion and/or Minion::Backend::SQLite:
```
****************************************************************
Regression 'mod:Minion::Backend::SQLite'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   1.65
[1='eq_0.006'] 	      0.0000	      0.0000	   1.82
[2='eq_4.002'] 	      1.0000	      0.0000	3369072521444784.50
[3='eq_4.004'] 	      1.0000	      0.0000	3690634035732229.50
[4='eq_4.005'] 	      1.0000	      0.0000	3862382255694749.50
[5='eq_v5.0.1']	     -0.0000	      0.0000	  -2.70

R^2= 1.000, N= 96, K= 6
****************************************************************
(4)
****************************************************************
Regression 'mod:Minion'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	27566166103309496.00
[1='eq_10.06'] 	     -1.0000	      0.0000	-4386092848057193.50
[2='eq_10.07'] 	     -1.0000	      0.0000	-4386092848057191.50
[3='eq_10.08'] 	     -1.0000	      0.0000	-8456913711195809.00
[4='eq_5.09']  	     -1.0000	      0.0000	-5338165111845252.00
[5='eq_9.09']  	      0.0000	      0.0000	   1.36
[6='eq_9.10']  	      0.0000	      0.0000	   0.51
[7='eq_9.13']  	      0.0000	      0.0000	   0.82

R^2= 1.000, N= 96, K= 8
****************************************************************
```

**Comment by eserte** [comment]
With more reports it seems that the only candidate is Minion (fails with >=10.06):
```
****************************************************************
Regression 'mod:Minion'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	28587076671297240.00
[1='eq_10.05'] 	      0.0000	      0.0000	   0.91
[2='eq_10.06'] 	     -1.0000	      0.0000	-4548531415828120.00
[3='eq_10.07'] 	     -1.0000	      0.0000	-4548531415828118.00
[4='eq_10.08'] 	     -1.0000	      0.0000	-11212773983347514.00
[5='eq_5.09']  	     -1.0000	      0.0000	-5535863593234436.00
[6='eq_9.09']  	      0.0000	      0.0000	   1.41
[7='eq_9.10']  	      0.0000	      0.0000	   0.53
[8='eq_9.13']  	      0.0000	      0.0000	   0.85

R^2= 1.000, N= 104, K= 9
****************************************************************
```

**Comment by cromedome** [comment]
This should be resolved by #3. Closing. Thanks for reporting!
